PowerShell/HttpTrigger/run.ps1

```diff
@@ -0,0 +1,38 @@
+using namespace System.Net
+
+# Input bindings are passed in via param block.
+param($Request, $TriggerMetadata)
+
+# Write to the Azure Functions log stream.
+Write-Host "PowerShell HTTP trigger function processed a request."
+
+# Local execution setting in local.settings.json
+# Azure execution setting configuration -> Application settings
+$slackWebhook = $env:SLACK_WEBHOOK_URL
+
+if ($Request.Body) {
+    $slackMessage = Get-SlackFormatMessage -Payload $Request.Body
+    $postResponse = Send-SlackMessage -SlackMessage $slackMessage -SlackUrl $slackWebhook
+
+    if ($postResponse.StatusCode -eq 200) {
+        Write-Host("Webhook successfully sent to Slack")
```

Comment:
Powershell doesn't use parenthesis to call functions

Reply:
Fixed.